ddtrace/opentracer/span.py

```diff
@@ -123,11 +123,11 @@ def log_kv(self, key_values, timestamp=None):
                 self._dd_span.error = 1
                 self.set_tag("error", 1)
             elif key == "error" or key == "error.object":
-                self.set_tag(ERROR_TYPE, val)
+                self.set_tag_str(ERROR_TYPE, val)
```

Comment:
I don't think we need to update the set_tag attribute on opentracing spans. We should only convert `self. _dd_span.set_tag()` to `self. _dd_span.set_tag_str()`